.github/workflows/create_pr_for_changelog.yml

```diff
@@ -0,0 +1,52 @@
+name: Generate changelog PR
+
+on:
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Generate the full changelog
+        uses: orhun/git-cliff-action@v4
+        id: git_cliff
+        with:
+          config: cliff.toml
+          args: --verbose --exclude-path "data/" --bump
+        env:
+          OUTPUT: CHANGELOG.md
+
+      - name: Generate latest changes
+        uses: orhun/git-cliff-action@v4
+        id: git_cliff_latest
+        with:
+          config: cliff.toml
+          args: --verbose --latest --strip header --exclude-path "data/" --bump --unreleased
```

Comment:
Not entirely sure about excluding `data/` entirely as we may still want to list data updates in our changelog; what do @ccomb and @paulboosz think?

Reply:
I'd like to include changes in data but, as they are somehow different, what about including them in a dedicated data section (like features, bug fixes) ?

Reply:
The data generation pipeline and experiments is of no interest to end users. Actually I'm in favor of moving `data/` into the ecobalyse-data repo.

Reply:
This is something we could change latter on. I'm in favor of merging this PR as it is to be able to use it for the next release.